src/llama_stack_client/lib/cli/llama_stack_client.py

```diff
@@ -34,14 +34,12 @@ def cli(ctx, endpoint: str, config: str | None):
     ctx.ensure_object(dict)
 
     # If no config provided, check default location
-    if config is None:
+    if config:
         if endpoint != "":
             raise ValueError("Cannot use both config and endpoint")
         default_config = get_config_file_path()
         if default_config.exists():
```

Comment:
Do we want to override original config when using `--config`?

Reply:
ya, fixed it.